app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,14 @@
+class FoodsController < ApplicationController
+
+  def index
+    @results = FoodSearchFacade.search_foods(food_params[:q])
```

Comment:
Since we aren't trying to write to our db, we don't need to use safe params here. You should be able to pass `params[:q]` directly.